solana-programs/start.sh

```diff
@@ -137,13 +139,33 @@
     anchor deploy --provider.cluster $SOLANA_HOST
 
     # Initialize Audius Admin account
-    yarn run ts-node cli/main.ts -f initAdmin -k ~/.config/solana/id.json -n $SOLANA_HOST
+    yarn run ts-node cli/main.ts -f initAdmin -k $owner_wallet_path -n $SOLANA_HOST
 
     # Propagate local variables
     admin_keypair_publickey=$(solana-keygen pubkey adminKeypair.json)
     admin_keypair_privatekey=$(cat adminKeypair.json)
     admin_storage_keypair_publickey=$(solana-keygen pubkey adminStorageKeypair.json)
     admin_storage_keypair_privatekey=$(cat adminStorageKeypair.json)
+
+    # initialize Content/URSM nodes - initContentNode uses deterministic 
+    # addresses and pkeys from eth-contracts ganache chain.
+    yarn run ts-node cli/main.ts -f initContentNode \
```

Comment:
here is the meat of this - since we're using deterministic addresses for local dev we can init in start.sh before CNs are even up bc we already know what the ganache accounts will be.